src/components/order-summary/order-summary-category.html

```diff
@@ -0,0 +1,90 @@
+          <li data-option data-value="all" class="bx--dropdown-item"><a class="bx--dropdown-link" href="javascript:void(0)">USD</a></li>
+          <li data-option data-value="cloudFoundry" class="bx--dropdown-item"><a class="bx--dropdown-link" href="javascript:void(0)">NOK</a></li>
+          <li data-option data-value="staging" class="bx--dropdown-item"><a class="bx--dropdown-link" href="javascript:void(0)">GBP</a></li>
+          <li data-option data-value="dea" class="bx--dropdown-item"><a class="bx--dropdown-link" href="javascript:void(0)">EUR</a></li>
+        </ul>
+      </li>
+    </ul>
+
+  </section>
+  <ul class="bx--order-list">
+    <li class="bx--order-category">
+      <p class="bx--order-category-title">Category Label</p>
+      <ul>
+        <li class="bx--order-item">
+          <p class="bx--order-detail">Detail one</p>
+          <p class="bx--order-price">$10.00 /mo</p>
+        </li>
+        <li class="bx--order-item">
+          <p class="bx--order-detail">Detail one</p>
+          <p class="bx--order-price">$20.00</p>
+        </li>
+        <li class="bx--order-item">
+          <p class="bx--order-detail">Detail one</p>
+          <p class="bx--order-price">$20.00</p>
+        </li>
+      </ul>
+    </li>
+    <li class="bx--order-category">
+      <p class="bx--order-category-title">Category Label</p>
+      <ul>
+        <li class="bx--order-item">
+          <p class="bx--order-detail">Extra long detail to show what a long line is like</p>
+          <p class="bx--order-price">$20000.00</p>
+        </li>
+        <li class="bx--order-item">
+          <p class="bx--order-detail">Detail one</p>
+          <p class="bx--order-price">$20.00</p>
+        </li>
+        <li class="bx--order-item">
+          <p class="bx--order-detail">Detail one</p>
+          <p class="bx--order-price">$20.00</p>
+        </li>
+      </ul>
+    </li>
+    <li class="bx--order-category">
+      <p class="bx--order-category-title">Category Label</p>
+      <ul>
+        <li class="bx--order-item">
+          <p class="bx--order-detail">Detail one</p>
+          <p class="bx--order-price">$20.00</p>
+        </li>
+        <li class="bx--order-item">
+          <p class="bx--order-detail">Detail one</p>
+          <p class="bx--order-price">$20.00</p>
+        </li>
+        <li class="bx--order-item">
+          <p class="bx--order-detail">Detail one</p>
+          <p class="bx--order-price">$20.00</p>
+        </li>
+      </ul>
+    </li>
+  </ul>
+  <section class="bx--order-total-container">
+    <div class="bx--order-total">
+      <p class="bx--order-total-text">Total due now</p>
+      <p class="bx--order-total-price">
+        $0.00
+        <span>estimated</span>
+      </p>
+    </div>
+    <p class="bx--order-total-subtitle"> * This is an estimated price based on a customer’s average usage, includes monthly and hourly pricing.</p>
+    <button class="bx--btn bx--btn--primary" type="button">Create Cluster</button>
```

Comment:
@hellobrian switched this to `type="submit"` based on my conversation with @rizcheldayao

<img width="757" alt="screen shot 2017-07-11 at 2 40 35 pm" src="https://user-images.githubusercontent.com/11928039/28087449-1a07c8a6-6648-11e7-8f1b-4fad68c6a434.png">